### ISSUE TITLE
scriptioContinuaTokenization requirements.txt installation on Windows

### ISSUE THREAD

**Issue by Spidersouris** [hsnborn22/LinGo#8]
Hi! Thank you for this project.

I've been having some difficulties trying to install the requirements for scriptioContinuaTokenization on Windows (11). It seems that the requirements.txt file is not well suited for Windows as it makes pip throw a bunch of errors when running `pip install -r requirements.txt`.

In particular, some packages that can be found in the file are Ubuntu packages and cannot be installed on Windows, namely **apturl**, **Brlapi**, **command-not-found**, **cupshelpers** or **distro-info**.

There are also packages which version is wrong ([**defer** has no 1.0.6 release on pypi](https://pypi.org/project/defer/#history) for instance).

I've not yet tried installing it on my Ubuntu VM but I guess it'd be great if it could work from the get-go on Windows.

**Comment by hsnborn22** [comment]
I tried it and even though it displays some errors it should still install all the required packages; try to start one of the servers and see if it works.

**Comment by enrico3** [comment]
Using the Windows binary is not an option for you?

**Comment by Spidersouris** [comment]
Actually it seems most of the packages in the requirements aren't actually used by the app. I had not had time to take a look at the code when writing this but from what I have checked, the servers only use spaCy and the language-specific news corpora.

I tried starting the server on my Windows machine without installing all the packages and it seems to work fine. Did you simply `pip freeze -r > requirements.txt` to get the file? It may have included packages that are installed on the environment but not actually necessary for LinGo to run properly. It may be necessary to do a quick cleanup of the file; or, if some of the packages are actually required on Linux, perhaps create two different `requirements.txt` files for each OS?

The problem with the current instructions is that pip will fail as soon as it encounters a package that cannot be installed and the rest of the packages in `requirements.txt` will not be installed. There's a way to read each line of the file and proceed to the installation of each package individually though, if you want to add this to the doc:

`cat requirements.txt | xargs -n 1 pip install`

> Using the Windows binary is not an option for you?

According to the doc:

> It doesn't matter if you installed the app using the binaries or built it from source, if you want to make the "scriptio continua" languages work, you will have to run the respective server for the language in the scriptioContinuaTokenization folder

And then it proceeds to showing how to install the requirements. But perhaps it's the doc that's wrong :)
